tests/fixture/tmpnet/subnet.go

```diff
@@ -339,14 +339,10 @@ func WaitForActiveValidators(
 
 // Reads subnets from [network dir]/subnets/[subnet name].json
 func readSubnets(subnetDir string) ([]*Subnet, error) {
-	if _, err := os.Stat(subnetDir); os.IsNotExist(err) {
+	entries, err := os.ReadDir(subnetDir)
+	if os.IsNotExist(err) {
```

Comment:
I think the doc on `os.IsNotExist` says:

> // This function predates errors.Is. It only supports errors returned by
// the os package. New code should use errors.Is(err, fs.ErrNotExist).

Should we align the error checks to this new format (which is what we currently use in `readState`)